apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/OperationBuilder.kt

```diff
@@ -35,6 +35,7 @@ class OperationBuilder(
     private val generateQueryDocument: Boolean,
     private val operation: IrOperation,
     flatten: Boolean,
+    private val useJvmAnnotations: Boolean = false,
```

Comment:
We'll need this to be available in `FragmentBuilder`. It's a lot more niche but it's used when reading fragments from the cache.